addons/website_slides_survey/tests/test_course_certification_unlink.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import tagged
+from odoo.tests.common import  TransactionCase
+from odoo.exceptions import ValidationError, UserError
+
+
+@tagged('unlink')
+class TestCourseCertifUnlink(TransactionCase):
```

Comment:
Maybe rename the file `TestSurvey` and inherit from `TestSurveyCommon`

The file of a test should not be specific to a "particular action" (here unlink)